stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing_view.xml

```diff
@@ -84,4 +84,42 @@
         <field name="binding_model_id" ref="stock.model_stock_move_line" />
         <field name="target">new</field>
     </record>
+    <record id="act_stock_barcode_selection_printing_quant" model="ir.actions.server">
+        <field name="name">Print barcode labels</field>
+        <field name="model_id" ref="stock.model_stock_quant" />
+        <field name="binding_model_id" ref="stock.model_stock_quant" />
+        <field name="binding_type">report</field>
+        <field name="state">code</field>
+        <field name="code">
+lines = []
```

Comment:
This bunch of code is very likely to fail without tests. It's even not very recommended to add it this way. Can you add at least a test or convert it to other technique?